core-common/src/main/java/org/zalando/nakadi/repository/kafka/KafkaSettings.java

```diff
@@ -30,7 +31,8 @@ public KafkaSettings(@Value("${nakadi.kafka.request.timeout.ms}") final int requ
                          @Value("${nakadi.kafka.enable.auto.commit}") final boolean enableAutoCommit,
                          @Value("${nakadi.kafka.max.request.size}") final int maxRequestSize,
                          @Value("${nakadi.kafka.delivery.timeout.ms}") final int deliveryTimeoutMs,
-                         @Value("${nakadi.kafka.max.block.ms}") final int maxBlockMs) {
+                         @Value("${nakadi.kafka.max.block.ms}") final int maxBlockMs,
+                         @Value("${nakadi.kafka.client.rack:}") final String clientRack) {
```

Comment:
is empty string a default value in the Kafka client?

Reply:
yes, it is